src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -158,9 +159,25 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
   // ads
   const { enabled: adsEnabled } = useToggle('ads');
   const { enabled: podcastPromoEnabled } = useToggle('podcastPromo');
-  const { isAmp, showAdsBasedOnLocation } = useContext(RequestContext);
+  const { isAmp, showAdsBasedOnLocation, mvtExperiments } =
+    useContext(RequestContext);
+  const { optimizely } = useContext(OptimizelyContext);
   const adcampaign = path(['metadata', 'adCampaignKeyword'], pageData);
 
+  const hasFullStackExperiment =
```

Comment:
Is this what controls the component variation to show the user?

Assume we have an experiment with variations, does "hasFullStackExperiment" need to be created for every variation? e.g. hasFullStackExperimentControl = ... hasFullStackExperimentVariation1...
If that is the case maybe this can be a function.